python/nav/web/templates/ipam/index.html

```diff
@@ -27,7 +26,7 @@
   <div id="prefix-tree">
     <div class="content">
     </div>
-    <!-- container for prefix tree -->
+ test_   <!-- container for prefix tree -->
```

Comment:
Sorry, what?

Reply:
This was pycharm freezing on me, and inputting the text in the file instead of the action menu :') It really loves eating up all the ram and behaving irrationally. I'll fix.